src/Products/CMFPlone/controlpanel/browser/configure.zcml

```diff
@@ -348,4 +348,13 @@
       permission="cmf.ManagePortal"
       />
 
+  <!-- RecycleBin -->
+
+  <browser:page
+      name="plone-recyclebin"
```

Comment:
This should be named recyclebin-controlpanel for consistency with other control panels.

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4223/commits/3c2b0eee7012b6ee4b5fc6009662cb0089adcccf